### PR TITLE
Proposed fix for ./log module not found

### PR DESCRIPTION
**Pull request by johan-lejdung**
**What kind of change does this PR introduce?**
Bugfix of https://github.com/webpack/webpack-dev-server/issues/1049

**Did you add or update the `examples/`?**
No

**Summary**
This is a proposed fix of the https://github.com/webpack/webpack-dev-server/issues/1049
Instead of just assuming that the module ./log is loaded there is a check to see if it actually is.

**Does this PR introduce a breaking change?**
It should not!

**Other information**

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ var onSocketMsg = {
 	},
 	"log-level": function(level) {
 		var hotCtx = require.context("webpack/hot", false, /^\.\/log$/);
-		if(hotCtx.keys().length > 0) {
+		if(hotCtx.keys().length > 0 && hotCtx.keys()["./log"]) {
```
Comment (shellscape) [inline client/index.js]: I'd like to see a small modification here please. 

```js
var contextKeys = hotCtx.keys();
if(contextKeys.length && contextKeys["./log"]) {
```

**Comment by shellscape** [comment]
@johan-lejdung thanks for the PR! we have one small change request and then we'll be happy to merge.

File: client/index.js (hunk 1 above)
Comment (johan-lejdung) [inline client/index.js]: Absolutely!

**Comment by shellscape** [comment]
Thanks!

**Pull request merged**